easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.1.1-centos6_x86_64.eb

```diff
@@ -3,8 +3,10 @@ version = '7.1.1'
 versionsuffix = '-centos6_x86_64'
 
 homepage = 'https://surfer.nmr.mgh.harvard.edu/'
-description = """FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
-FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data."""
+description = """FreeSurfer is a set of tools for analysis and visualization 
+of structural and functional brain imaging data.  FreeSurfer contains a fully 
+automatic structural imaging stream for processing cross sectional and 
+longitudinal data."""
```

Comment:
Note that this changes the module help text. I used the following for IMO more sensible line breaks in #21405

```
description = """\
FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data.
"""
```

If that sounds useful I can do the Search&replace using the texts I previously used.